src/containers/WatchList/DeleteMovie/container.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Modal } from 'antd'
+import { connect } from 'react-redux'
+import { openNotification } from 'Utils'
+
+import { removeMovieErrorsSelector } from 'Store/watchList/selectors'
+import {
+  removeMovie as removeMovieAction,
+  cleanupRemoveMovieErrors as cleanupRemoveMovieErrorsAction
+} from 'Store/watchList/actions'
+import DeleteMovieComponent from './component'
+
+class DeleteMovie extends Component {
+  onDeleteMovie = () => {
+    const { removeMovie, movieId, title } = this.props
+    return Modal.confirm({
+      title: 'Do you want to delete movie from watchlist?',
+      onOk() { removeMovie(movieId, title) }
+    })
+  }
+
+  render() {
+    const {
+      movieId,
+      title,
+      removeMovieErrors,
+      cleanupRemoveMovieErrors
+    } = this.props
+    const config = {
+      message: 'Error',
+      description: `Cant remove "${title}" from Watchlist`,
```

Comment:
Cant remove "${title}" from ***the*** Watchlist

Reply:
✅